candidategenerator.js

```diff
@@ -98,10 +172,10 @@ window.GEN = function(sTree, words, options){
 	return candidates;
 }
 
-function iotaIsHeaded(iota) {
+function rootIsHeaded(iota, recCat) {
 	var children = iota.children || [];
 	for (var i = 0; i < children.length; i++)
-		if (children[i].cat === 'phi')
+		if (children[i].cat === recCat)
```

Comment:
If someone enters 'i' for the root category and 'w' for the recursive category, then this will yield the wrong results, since we'd be looking for a 'w' to be the head of the 'i'. I think we should be checking for a child of category pCat.nextLower(iota.cat) [except that I want iota to get renamed].

Reply:
recCat replaced with pCat.nextlower(pRoot.cat)